test/sparse/sparse.jl

```diff
@@ -1,469 +1,518 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-@test issparse(sparse(ones(5,5)))
-@test !issparse(ones(5,5))
-@test Base.SparseArrays.indtype(sparse(ones(Int8,2),ones(Int8,2),rand(2))) == Int8
-
-# check sparse matrix construction
-@test isequal(Array(sparse(complex(ones(5,5),ones(5,5)))), complex(ones(5,5),ones(5,5)))
-@test_throws ArgumentError sparse([1,2,3], [1,2], [1,2,3], 3, 3)
-@test_throws ArgumentError sparse([1,2,3], [1,2,3], [1,2], 3, 3)
-@test_throws ArgumentError sparse([1,2,3], [1,2,3], [1,2,3], 0, 1)
-@test_throws ArgumentError sparse([1,2,3], [1,2,3], [1,2,3], 1, 0)
-@test_throws ArgumentError sparse([1,2,4], [1,2,3], [1,2,3], 3, 3)
-@test_throws ArgumentError sparse([1,2,3], [1,2,4], [1,2,3], 3, 3)
-@test isequal(sparse(Int[], Int[], Int[], 0, 0), SparseMatrixCSC(0, 0, Int[1], Int[], Int[]))
-@test sparse(Any[1,2,3], Any[1,2,3], Any[1,1,1]) == sparse([1,2,3], [1,2,3], [1,1,1])
-@test sparse(Any[1,2,3], Any[1,2,3], Any[1,1,1], 5, 4) == sparse([1,2,3], [1,2,3], [1,1,1], 5, 4)
-
-# check matrix operations
+@testset "issparse" begin
+    @test issparse(sparse(ones(5,5)))
+    @test !issparse(ones(5,5))
+    @test Base.SparseArrays.indtype(sparse(ones(Int8,2),ones(Int8,2),rand(2))) == Int8
+end
```

Comment:
No, sorry, github associated this comment with the wrong line. Kindly disregard my comments for the moment; chances are they will not be comprehensible without correct line association.